### PR TITLE
README Revision

### PR DESCRIPTION
**Pull request by uoduckswtd1**
Proposed revision to README file, in alignment with text in shared google doc reviewed earlier (https://docs.google.com/document/d/1pVtYHNTwrGFV3UyiR2qBKV0MvipoEEI-fbwavSNui4Q/edit).

### CODE REVIEW

**Comment by GeorgLink** [comment]
Thank you very much @uoduckswtd1 and congratulations to mastering the pull request system 👍

File: README.md [hunk 1]
```diff
@@ -155,19 +172,10 @@ Are you eligible to be on this list? You are if you helped in any capacity, for
 Created a Pull Request. Gave feedback on our work. The team will try to update this list monthly,
 but please open an issue or post on the mailing list if we've missed anyone.
 
-
 If you find yourself missing, please create a pull request or reach out to a maintainer. We started to maintain this list after starting the working group and are likely missing some of you. If you find yourself listed here and want to be removed, please create a pull request or ask a maintainer.
 
-## Related Works
+## License
+
+Text ([related issue #149](https://github.com/chaoss/wg-diversity-inclusion/issues/149))
```
Comment (GeorgLink) [inline README.md]: Line 179 should link to our license file and name our license.

I suggest MIT, in anticipation of merging #169 soon.

Comment (uoduckswtd1) [reply README.md]: Thanks @GeorgLink, I'm getting the hang of it! Submitted revised update to line #179, license.

**Comment by GeorgLink** [comment]
We can merge this after #169 is merged.

**Pull request merged**